tests/test_definitions/column_aggregate_expectations/expect_column_mean_to_be_between.json

```diff
@@ -53,6 +58,20 @@
             "observed_value": 3.5
           }
         },
+        {
```

Comment:
@TrangPham Side note: If we indeed find that decimals are supported by some backends, while not by others, we will have to use `only_for` / `suppress_for` in these `JSON` test cases and/or potentially add `JSON` files for specific backends (we can figure this out next week after we understand why decimals are not supported for certain backends).  Thank you.

Reply:
Great point, would be happy to pair on this if you need.

Reply:
For some reason, including these test cases here pass when I don't think they should have. The decimal test case for spark was passing before I added my new code.

Reply:
@TrangPham Are we sure that they even run?

Reply:
How would I check on that?